lualib/resty/events/init.lua

```diff
@@ -81,13 +55,35 @@ function _M.configure(opts)
         return nil, '"unique_timeout" must be greater than 0'
     end
 
-    local is_broker = worker_id == opts.broker_id
+    return true
+end
+
+function _M.new(opts)
+    local ok, err = check_options(opts)
+    if not ok then
+        return nil, err
+    end
```

Comment:
`assert` here perhaps?

```suggestion
assert(check_options(opts))
```

Reply:
We want to return an error message to the client, so won't use `assert`.

Reply:
done.